pkg/terraform/stages/kubevirt/stages.go

```diff
@@ -0,0 +1,14 @@
+package kubevirt
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/stages"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in
+// Kubevirt.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage("kubevirt", "datavolume"),
+	stages.NewStage("kubevirt", "masters"),
```

Comment:
What is the reason for separating datavolume and masters into different stages? Can they not be combined into one "cluster" stage?

Reply:
No reason other than  ignorance on my part. It was previously configured this way as a module so I separated it out this way. I don't see any reason why it can't be combined.

Reply:
I have updated this.